src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessarysemicolonintrywithresources/Example2.java

```diff
@@ -0,0 +1,29 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="UnnecessarySemicolonInTryWithResources">
+      <property name="allowWhenNoBraceAfterSemicolon" value="false"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.unnecessarysemicolonintrywithresources;
+
+import java.io.IOException;
+import java.io.PipedReader;
+import java.io.Reader;
+
+// xdoc section -- start
+class Example2 {
+  void method() throws IOException {
```

Comment:
Please move `// xdoc section -- start` under 
Class, to not print class name in xdoc.
Method is all that user needs to see to understand how it works

Reply:
Hey @romani, when I move it under the Class I am getting the following error when runnin `mvn clean verify`
`/unnecessarysemicolonintrywithresources.xml:52: Content of source tag should not be Indented [noIndentationConfigExamples]` (for both examples). Even when I change the indentation to the xml file it somehow auto-idents back. Any pointers on how to overcome this?

Reply:
hmm, strange, I thought we did support of this, but looks like we missed....

ok, I recall my review request.

Reply:
The only other place I can see the xdoc to be under the class is in [javadocblocktaglocation](https://github.com/checkstyle/checkstyle/blob/474873ce4d7e632822155b6a4cba7dbda882d2a1/src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocblocktaglocation/Example1.java#L11-L19), but the indentation here is kind of different as it's not a method, this might be the reason it works there but not sure how exactly it works